coding-standards/git.md

```diff
@@ -49,7 +52,20 @@ Pull Requests (PRs):
 
 -   Create PRs for merging changes into the `main` or `development` branches.
 -   Ensure PRs are reviewed and approved by peers before merging.
--   Provide a detailed description in the PR, explaining the changes and their purpose (why?).
+-   Provide a detailed description in the PR. Use the body to explain what is it, why is it needed and how is it done,
+> \<What is it \>
+> \<Why is it needed\>
+> \<How did we achieve this\>
```

Comment:
![image](https://github.com/OsmosysSoftware/dev-standards/assets/16570290/ce5dce1a-2487-4f78-8fea-cb6e3cd6c0a0)

I don't think we need all these 3 notes since they are already covered in the above point. just end the above point with a period. If we need then add them in a code block if required and format them better.